src/backend/joanie/tests/payment/test_models_invoice.py

```diff
@@ -208,7 +208,11 @@ def test_models_invoice_relying_on_order_with_a_negative_amount(self):
 
         self.assertEqual(
             str(context.exception),
-            "{'__all__': ['Credit note must have a parent invoice.']}",
+            (
+                "{'__all__': ["
+                "'Constraint “main_invoice_should_have_a_positive_amount” is violated.'"
```

Comment:
why remove the separate message when parent is not set? It's a bit convoluted to understand that parent is not set when I get this error.